### ISSUE TITLE
Candidate evaluation period

### ISSUE THREAD

**Issue by yourheropaul** [Higher-labs/higher#15]
I propose that the candidate evaluation time should necessarily be equal to the employer's company's probation period.

Instead, there should be a _candidate evaluation period_, set by the Higher system, in which the employer must evaluate the endorsed hard and soft skills of the candidate they hired. At the end of this period, even if it's shorter than the company probation period, the Higher process will conclude (see #8 ).

I'm imagining a ballpark duration of 6 weeks, but that was arbitrarily chosen. Perhaps it should be a function of the number and level of required skills.

The reasons for this candidate evaluation period are:

1. Higher provides value when it provides a suitable candidate. Suitability is defined by the candidate having skills that equal or exceed the role's required skills. When provided, the hiring process should be concluded for that particular role and opening. 

    In other words, we were thinking that backers endorse the candidate passing the employer's probation; I propose that backs endorse the candidate being _able_ to pass probation.

2. The candidate can also 'cancel' by quitting. They possess all the required skills, but may not like the job, or they may be overqualified. After accepting a job offer, Higher should require them to stick it out for at least the minimum evaluation period, but it shouldn't require more than that.
